src/hotspot/share/gc/g1/g1BlockOffsetTable.cpp

```diff
@@ -176,10 +163,10 @@ void G1BlockOffsetTablePart::check_all_cards(uint8_t* start_card, uint8_t* end_c
 //       ( ^    ]
 //         blk_start
 //
-void G1BlockOffsetTablePart::update_for_block_work(HeapWord* blk_start,
+void G1BlockOffsetTable::update_for_block_work(HeapWord* blk_start,
```

Comment:
Some indentation issues.

Reply:
Fixed. I tried to find other indentation issues except this one, but can't find now. Please point it out if you found.